src/app/components/NetworkRibbon/NetworkRibbon.tsx

```diff
@@ -0,0 +1,50 @@
+import { useWalletContext, walletService } from '@sovryn/react-wallet';
+import { ProviderType, web3Wallets } from '@sovryn/wallet';
+import React, { useEffect, useState } from 'react';
+
+import { currentChainId, currentNetwork } from '../../../utils/classifiers';
+import { Dialog } from '../../containers/Dialog/Loadable';
+
+import './_networkRibbon.scss';
+
+export function NetworkRibbon() {
+  const { connected, wallet } = useWalletContext();
+  const getStatus = () =>
+    connected &&
+    web3Wallets.includes(wallet.providerType) &&
+    wallet.chainId !== currentChainId;
+  const [isConnect, setShow] = useState(getStatus());
```

Comment:
variable `isConnect` is not informative, something like [`isOpen`, setOpen] would make more sense.